docs/quest_poi_points.md

```diff
@@ -8,28 +8,163 @@ Comes from sniffs. Visually speaking, this table is used to identify the X and Y
 
 **Structure**
 
-|                                      |          |                |         |          |             |           |             |
-|--------------------------------------|----------|----------------|---------|----------|-------------|-----------|-------------|
-| **Field**                            | **Type** | **Attributes** | **Key** | **Null** | **Default** | **Extra** | **Comment** |
-| [questid](#quest_poi_points-questid) | int(10)  | unsigned       | PRI     | NO       | 0           |           |             |
-| [id](#quest_poi_points-id)x1         | int(10)  | unsigned       | PRI     | NO       | 0           |           |             |
-| [idx](#quest_poi_points-idx)2        | int(10)  | unsigned       | PRI     | NO       | 0           |           |             |
-| [x](#quest_poi_points-x)             | int(10)  | unsigned       |         | NO       | 0           |           |             |
-| [y](#quest_poi_points-y)             | int(10)  | unsigned       |         | NO       | 0           |           |             |
+<table>
+<tbody>
+<tr class="even">
+<td><p><strong>Field</strong></p></td>
+<td><p><strong>Type</strong></p></td>
+<td><p><strong>Attributes</strong></p></td>
+<td><p><strong>Key</strong></p></td>
+<td><p><strong>Null</strong></p></td>
+<td><p><strong>Default</strong></p></td>
+<td><p><strong>Extra</strong></p></td>
+<td><p><strong>Comment</strong></p></td>
+</tr>
+<tr class="even">
+<td><p><a href="#quest_poi_points-questid">questid</a></p></td>
+<td><p>int(10)</p></td>
+<td><p>unasigned</p></td>
+<td><p>PRI</p></td>
+<td><p>NO</p></td>
+<td><p>0</p></td>
+<td><p></p></td>
+<td><p></p></td>
+</tr>
+<tr class="even">
+<td><p><a href="#quest_poi_points-idx">idx</a>2</p></td>
```

Comment:
the 2 is not inside the <a> https://github.com/azerothcore/wiki/blob/137f604de4cb53638b8642bc6da9ff2bcc202a23/docs/quest_poi_points.md

Reply:
I don't understand